metadata.yaml

```diff
@@ -63,7 +63,11 @@ provides:
     description: |
       Forwards the built-in grafana dashboard(s) for monitoring traefik.
     interface: grafana_dashboard
-
+  tracing:
+    description: |
+      Tracing endpoint for integrating with tempo.
+    limit: 1
```

Comment:
was not aware of the 'server should be provider' discussion. This will require changing the directionality of the tracing relation, submitting a new pr to CRI, etc...

Reply:
relating to multiple tempo apps is not supported by traefik. We can configure traefik with ONE tracing endpoint to send traces to.

I don't see how tempo having multiple units is of interest to Traefik, can you explain?

Reply:
> how tempo having multiple units is of interest to Traefik

Just confirming if there are any assumptions in the code, e.g. how we get/iterate relations.